trpc/router/mailbox/index.ts

```
@@ -63,6 +63,59 @@ export const mailboxRouter = {
+            eq(conversations.status, status),
+            isNull(conversations.mergedIntoId),
+            where,
+          ),
+        );
+      return result[0]?.count ?? 0;
+    };
+
+    const [
+      openAll, openMine, openAssigned,
+      closedAll, closedMine, closedAssigned,
+      spamAll, spamMine, spamAssigned
```

ESLint

[error] Replace `⏎······openAll,·openMine,·openAssigned,⏎······closedAll,·closedMine,·closedAssigned,⏎······spamAll,·spamMine,·spamAssigned` with `openAll,·openMine,·openAssigned,·closedAll,·closedMine,·closedAssigned,·spamAll,·spamMine,·spamAssigned]·=`

(prettier/prettier)